README.rst

```diff
@@ -80,3 +80,15 @@ If your predicate might raise certain exceptions you wish to ignore, you may use
  True
  >>> wait(predicate, expected_exceptions=(ValueError, AttributeError))
  True
+
+If you'd like to maintain updates while waiting for a predicate to complete, you may use ``update_msg`` to pass a lambda to log the status. By default, this is a no-op.
+
+  >>> import logging
+  >>> from waiting import wait
+  >>> try:
+  ...    wait(lambda: False, timeout_seconds=5,               # Timeout after 5 seconds
+  ...          update_msg=lambda: logging.warn("Waiting...")) # Log "Waiting..." six times.
```

Comment:
update_msg is not a good name for "just a function that gets called on every poll loop".

I think "on_poll" is a much better name